backend/.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
+.env
```

Comment:
Do we need this file in the `backend` folder?
We have a file like this at the top level